.github/workflows/schedule.yml

```diff
@@ -16,10 +16,10 @@ jobs:
           DJANGO_SU_NAME: "admin"
           DJANGO_SU_EMAIL: "admin@localhost.company"
           DJANGO_SU_PASSWORD: "12qwaszx"
-          API_ABOUT_PAGE: "localhost:8080/api/v1/server/about"
+          API_ABOUT_PAGE: "localhost:8080/api/server/about"
         run: |
           docker-compose -f docker-compose.yml -f docker-compose.dev.yml -f ./tests/docker-compose.email.yml -f tests/docker-compose.file_share.yml -f components/serverless/docker-compose.serverless.yml up -d --build
-          /bin/bash -c 'while [[ "$(curl -s -o /dev/null -w ''%{http_code}'' ${API_ABOUT_PAGE})" != "401" ]]; do sleep 5; done'
+          /bin/bash -c 'while [[ $(curl -s -o /dev/null -H "Accept: application/vnd.cvat+json; version=1.0" -w "%{http_code}" ${API_ABOUT_PAGE}) != "401" ]]; do sleep 5; done'
```

Comment:
@Marishka17 , what do you think if by default we will use the latest version of API? Thus if we don't specify accept header, DRF will use the latest one. I know that the link https://github.com/interagent/http-api-design/blob/master/en/foundations/require-versioning-in-the-accepts-header.md says that it is a bad practice. But GitHub uses the strategy: https://docs.github.com/en/rest/overview/media-types

Reply:
@nmanovic ,  Also rest_framework [documentation](https://www.django-rest-framework.org/api-guide/versioning/) doesn't say that this is a bad practice, let's use the default version. In this case, I guess we can miss `accept header` in client/tests. I'll update it.